frontend/src/app/application-forms/tree-application-form/tree-permit-view/tree-permit-view.component.html

```diff
@@ -90,15 +90,15 @@ <h3>Your permit and the Christmas tree cutting rules and guidelines were emailed
   </div>
 
 
-  <div *ngIf="error && permit && forest">
-    <div id="pay-gov-errors" class="usa-alert usa-alert-error" aria-live="assertive" aria-hidden="false" role="alert">
+  <div *ngIf="error && permit && forest" class="usa-width-one-whole margin-top-2">
+    <div id="pay-gov-errors" class="usa-alert usa-alert-error" aria-live="assertive" aria-hidden="false" role="alert" aria-label="error">
       <div class="usa-alert-body">
         <h3 class="usa-alert-heading">Unable to process your permit. Your credit card has not been charged.</h3>
         <p class="usa-alert-text">{{error.message}}</p>
       </div>
     </div>
     <div class="margin-top-2 margin-bottom-2">
-      <p *ngIf="forest"><strong>Return to the <a [href]="'/christmas-trees/forests/' + forest.forestAbbr + '/applications/' + permit.permitId">{{forest.forestName}} Christmas tree permit purchase form</a> and try again.</strong></p>
+      <p *ngIf="forest"><strong>Return to the <a [href]="'/christmas-trees/forests/' + forest.forestAbbr + '/applications/'">{{forest.forestName}} Christmas tree permit purchase form</a> and try again.</strong></p>
```

Comment:
If this is expecting javascript between the `"`s then can you use string interpolation with backticks here?

Reply:
I just need to verify that angular templating can handle back ticks and then will change.